daemon/inspect.go

```diff
@@ -151,3 +166,30 @@ func (daemon *Daemon) VolumeInspect(name string) (*types.Volume, error) {
 	}
 	return volumeToAPIType(v), nil
 }
+
+func getBackwardsCompatibleNetworkSettings(settings *network.Settings) *v1p20.NetworkSettings {
```

Comment:
@tiborvass Is this intentional to only show the bridge network settings(if container is connected to default bridge network) in the backward compatible fields? What will be the values of these fields if the container is connected to a non-default network and only to that network? My reading of the code says that these fields will be empty in that case because we have removed the initialization of  backward compatible fields in `container.NetworkSettings`